NewsFeed/views/CCBRNewsQuickViewController.m

```diff
@@ -95,7 +107,13 @@ - (void)setViewModel:(CCBRNewsQuickViewModel *)viewModel {
 - (void)updateUI {
     self.titleLabel.text = self.viewModel.title;
     self.domainLabel.text = self.viewModel.domain;
-//    self.faviconView.image = self.viewModel.favicon;
+    self.faviconView.image = self.viewModel.favicon;
```

Comment:
This is not enough to update the favicon when navigating to another website. Please see the screen recording below!

https://user-images.githubusercontent.com/1826162/103257912-b771b700-49c5-11eb-8a5b-0ac66d64d736.mov